documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -34,14 +34,16 @@ Default `20000` (20 seconds).
 Default `/strimzi/topics`.
 .. `STRIMZI_TLS_ENABLED` to enable TLS support for encrypting the communication with Kafka brokers.
 Default `true`.
+.. `STRIMZI_TLS_AUTH_ENABLED` to enable authenticated communication with Kafka brokers.
+Default `true`.
 .. `STRIMZI_TRUSTSTORE_LOCATION` to the path to the truststore containing certificates for enabling TLS based communication.
-Mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
+Mandatory only if TLS and authentication are enabled through `STRIMZI_TLS_ENABLED` and `STRIMZI_TLS_AUTH_ENABLED`.
 .. `STRIMZI_TRUSTSTORE_PASSWORD` to the password for accessing the truststore defined by `STRIMZI_TRUSTSTORE_LOCATION`.
-Mandatory only if TLS is enabled through `STRIMZI_TLS_ENABLED`.
+Mandatory only if TLS and authentication are enabled through `STRIMZI_TLS_ENABLED` and `STRIMZI_TLS_AUTH_ENABLED`.
```

Comment:
Same as below - `STRIMZI_TRUSTSTORE_LOCATION` and `STRIMZI_TRUSTSTORE_PASSWORD` should require only `STRIMZI_TLS_ENABLED`.